SuiteCareers/SuiteCareers/Models/User.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SuiteCareers.Models
+{
+    public class User
+    {
+        public long Id { get; set; }
+        [Key, DataType(DataType.EmailAddress)]
```

Comment:
Though email is unique, I recommend using a separate, dedicated Id field `UserId`

To guarantee that an email can't be used twice in the database use the Unique annotation
```
[Index(IsUnique=true)]
[DataType(DataType.EmailAddress)]
```